src/usr/local/www/interfaces.php

```diff
@@ -274,7 +274,10 @@ function remove_bad_chars($string) {
 		$pconfig['type6'] = "slaac";
 		break;
 	case "dhcp6":
-		$pconfig['dhcp6-duid'] = $wancfg['dhcp6-duid'];
+		if(empty($config['system']['global-v6duid'])) {
+			unset($wancfg['dhcp6-duid']);
+		}
```

Comment:
This shouldn't be done. Individual interface option should not be automatically changed in GUI if global option is not set.